Directory.Build.targets

```diff
@@ -8,28 +8,34 @@
 
   <Target Name="CopyNativeAssembiles" AfterTargets="CopyFilesToOutputDirectory">
     <PropertyGroup>
-		<LibPrefix Condition="'$(OS)' != 'Windows_NT'">lib</LibPrefix>
-		<LibExtension Condition="'$(OS)' == 'Windows_NT'">.dll</LibExtension>
-		<LibExtension Condition="'$(OS)' != 'Windows_NT'">.so</LibExtension>
-		<LibExtension Condition="$([MSBuild]::IsOSPlatform('osx'))">.dylib</LibExtension>
+      <LibPrefix Condition="'$(OS)' != 'Windows_NT'">lib</LibPrefix>
+      <LibExtension Condition="'$(OS)' == 'Windows_NT'">.dll</LibExtension>
+      <LibExtension Condition="'$(OS)' != 'Windows_NT'">.so</LibExtension>
+      <LibExtension Condition="$([MSBuild]::IsOSPlatform('osx'))">.dylib</LibExtension>
 
-		<TargetArchitecture Condition="'$(Platform)' == ''">x64</TargetArchitecture>
-		<NativeTargetArchitecture Condition="'$(NativeTargetArchitecture)' == ''">$(TargetArchitecture)</NativeTargetArchitecture>
-		<BinDir Condition="'$(BinDir)'==''">$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'artifacts', 'bin'))</BinDir>
-		<NativeOutputPath>$(BinDir)Native\$(NativeTargetArchitecture).$(Configuration)\</NativeOutputPath>
+      <BinDir Condition="'$(BinDir)'==''">$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'artifacts', 'bin'))</BinDir>
+      <NativeOutputPath>$(BinDir)Native\$(TargetArchitecture).$(Configuration)\</NativeOutputPath>
 
-		<Platform Condition="'$(Platform)'==''">AnyCPU</Platform>
-		<PlatformConfig>$(Platform).$(Configuration)</PlatformConfig>
-		<OutputPath Condition="'$(OutputPath)'==''">$(BinDir)$(MSBuildProjectName)\Debug</OutputPath>
-	</PropertyGroup>
+      <Platform Condition="'$(Platform)'==''">AnyCPU</Platform>
+      <PlatformConfig>$(Platform).$(Configuration)</PlatformConfig>
+      <OutputPath Condition="'$(OutputPath)'==''">$(BinDir)$(MSBuildProjectName)\Debug</OutputPath>
+	  </PropertyGroup>
 
 		<ItemGroup>
-		<NativeAssemblyReference>
-			<FullAssemblyPath>$(NativeOutputPath)$(LibPrefix)%(NativeAssemblyReference.Identity)$(LibExtension)</FullAssemblyPath>
-		</NativeAssemblyReference>
+      <NativeAssemblyReference>
+        <FullAssemblyPath>$(NativeOutputPath)$(LibPrefix)%(NativeAssemblyReference.Identity)$(LibExtension)</FullAssemblyPath>
+      </NativeAssemblyReference>
 		</ItemGroup>
 
-		<Copy SourceFiles = "@(NativeAssemblyReference->'%(FullAssemblyPath)')"
+    <PropertyGroup>
+      <ShouldCopyx64>false</ShouldCopyx64>
```

Comment:
I'm not sure I understand the value of these properties. What other target architectures do we have beside x86, x64, arm64, or arm?

These properties appear to only be used in the below Copy, and as far as I can tell, that condition is always true.

Reply:
Oh, I see now, this is to support copying `LdaNative` and `MatrixFactorizationNative` assemblies on arm, but not the other native files.

Reply:
Correct. I figured we could also use this same pattern when we get to blazer WASM. Though I guess we can't really copy any native files there... so maybe this was a bit overkill.